src/provider/application.js

```diff
@@ -72,7 +73,18 @@ class Application extends EventEmitter {
       const width = calculateWidth(this.resizeConfig.WidthCalculationMethod);
       this.JSONRPC.notification('resize', [null, `${width}px`]);
     } else {
-      const height = calculateHeight(this.resizeConfig.heightCalculationMethod);
+      let height = calculateHeight(this.resizeConfig.heightCalculationMethod);
+
+      if (this.targetNodes.length > 0) {
+        const maxTargetHeight = this.targetNodes
+          .map(selector => document.querySelector(selector))
+          .filter(node => node)
```

Comment:
rather than filtering these out, just have your `getOffsetToBody` return zero in that case.